creator-node/package.json

```diff
@@ -124,6 +128,7 @@
   },
   "standard": {
     "env": [
+      "node",
```

Comment:
noob question - what does this do

Reply:
adds global variable environments to standard lint so it doesn't complain. https://eslint.org/docs/latest/user-guide/configuring/language-options#specifying-environments